backend/apid/middlewares/authentication.go

```diff
@@ -1,38 +1,64 @@
 package middlewares
 
 import (
+	"context"
+	"errors"
 	"net/http"
+	"strings"
 
 	corev2 "github.com/sensu/sensu-go/api/core/v2"
 	"github.com/sensu/sensu-go/backend/apid/actions"
 	"github.com/sensu/sensu-go/backend/authentication/jwt"
+	"github.com/sensu/sensu-go/backend/store"
 )
 
 // Authentication is a HTTP middleware that enforces authentication
 type Authentication struct {
 	// IgnoreUnauthorized configures the middleware to continue the handler chain
 	// in the case where an access token was not present.
 	IgnoreUnauthorized bool
+	Store              store.Store
 }
 
 // Then middleware
 func (a Authentication) Then(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		ctx := r.Context()
-		tokenString := jwt.ExtractBearerToken(r)
-		if tokenString != "" {
-			token, err := jwt.ValidateToken(tokenString)
-			if err != nil {
-				logger.WithError(err).Warn("invalid token")
-				writeErr(w, actions.NewErrorf(actions.Unauthenticated, "invalid credentials"))
+		authHeader, ok := r.Header["Authorization"]
+		if ok && len(authHeader) >= 1 {
+			headerString := authHeader[0]
+
+			// if the auth header contains Bearer, continue with token auth
+			if strings.Contains(headerString, "Bearer ") {
```

Comment:
`strings.HasPrefix(headerString, "Bearer ")` would be more precise here, and similar below